listenbrainz/webserver/__init__.py

```diff
@@ -77,6 +78,12 @@ def gen_app(debug=None):
 
     In the Flask app returned, blueprints are not registered.
     """
+    # As early as possible, if debug is True, set the log level of our 'listenbrainz' logger to DEBUG
+    # to prevent flask from creating a new log handler
+    if debug is True:
+        logger = logging.getLogger('listenbrainz')
```

Comment:
one thought here is that we could put a comment in `listenbrainz/__init__.py` where the logger is initially created, informing that the level is set to DEBUG here in gen_app only if FLASK_DEBUG=1?